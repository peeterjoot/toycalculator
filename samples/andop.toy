//THIS IS A GENERATED TEST CASE (./bitwiseop.perl --and).  DO NOT EDIT

INT32 i;
INT64 r;
BOOL i1;
INT16 i16;
INT32 i32;
INT64 i64;
INT8 i8;
BOOL j1;
INT16 j16;
INT32 j32;
INT64 j64;
INT8 j8;
i1 = TRUE;
i16 = 1000;
i32 = 100000;
i64 = 100000000000;
i8 = 10;
j1 = FALSE;
j16 = 100;
j32 = 10000;
j64 = 10000000000;
j8 = 1;
i = 12340000;
PRINT i;
r = i1 AND j1;
PRINT r;
i = 12340001;
PRINT i;
r = i1 AND j16;
PRINT r;
i = 12340002;
PRINT i;
r = i1 AND j32;
PRINT r;
i = 12340003;
PRINT i;
r = i1 AND j64;
PRINT r;
i = 12340004;
PRINT i;
r = i1 AND j8;
PRINT r;
i = 12340005;
PRINT i;
r = i16 AND j1;
PRINT r;
i = 12340006;
PRINT i;
r = i16 AND j16;
PRINT r;
i = 12340007;
PRINT i;
r = i16 AND j32;
PRINT r;
i = 12340008;
PRINT i;
r = i16 AND j64;
PRINT r;
i = 12340009;
PRINT i;
r = i16 AND j8;
PRINT r;
i = 12340010;
PRINT i;
r = i32 AND j1;
PRINT r;
i = 12340011;
PRINT i;
r = i32 AND j16;
PRINT r;
i = 12340012;
PRINT i;
r = i32 AND j32;
PRINT r;
i = 12340013;
PRINT i;
r = i32 AND j64;
PRINT r;
i = 12340014;
PRINT i;
r = i32 AND j8;
PRINT r;
i = 12340015;
PRINT i;
r = i64 AND j1;
PRINT r;
i = 12340016;
PRINT i;
r = i64 AND j16;
PRINT r;
i = 12340017;
PRINT i;
r = i64 AND j32;
PRINT r;
i = 12340018;
PRINT i;
r = i64 AND j64;
PRINT r;
i = 12340019;
PRINT i;
r = i64 AND j8;
PRINT r;
i = 12340020;
PRINT i;
r = i8 AND j1;
PRINT r;
i = 12340021;
PRINT i;
r = i8 AND j16;
PRINT r;
i = 12340022;
PRINT i;
r = i8 AND j32;
PRINT r;
i = 12340023;
PRINT i;
r = i8 AND j64;
PRINT r;
i = 12340024;
PRINT i;
r = i8 AND j8;
PRINT r;
