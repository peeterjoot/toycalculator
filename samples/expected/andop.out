12340000
0
12340001
0
12340002
0
12340003
0
12340004
1
12340005
0
12340006
96
12340007
768
12340008
0
12340009
0
12340010
0
12340011
32
12340012
1536
12340013
99328
12340014
0
12340015
0
12340016
0
12340017
8192
12340018
9663864832
12340019
0
12340020
0
12340021
0
12340022
0
12340023
0
12340024
0
